src/ngx_http_lua_ssl_module.c

```diff
@@ -226,12 +226,25 @@ ngx_http_lua_ffi_update_ticket_encryption_key(SSL_CTX *ctx,
      * key. */
     if (keys->nelts > 0) {
         pkey = keys->elts;
-        if (ngx_memcmp(pkey->name, key, 16) == 0 &&
-            ngx_memcmp(pkey->aes_key, key + 16, 16) == 0 &&
-            ngx_memcmp(pkey->hmac_key, key + 32, 16) == 0)
-        {
-            dd("duplicate ticket key");
-            return NGX_OK;
+        if (ngx_strlen(key) == 48) {
+            dd("key size is 48");
+            if (ngx_memcmp(pkey->name, key, 16) == 0 &&
+                ngx_memcmp(pkey->aes_key, key + 16, 16) == 0 &&
+                ngx_memcmp(pkey->hmac_key, key + 32, 16) == 0)
+            {
+                    dd("duplicate ticket key");
+                    return NGX_OK;
+            }
+
+        } else if (ngx_strlen(key) == 80) {
+            dd("key size is 80");
+            if (ngx_memcmp(pkey->name, key, 16) == 0 &&
+                ngx_memcmp(pkey->aes_key, key + 16, 32) == 0 &&
```

Comment:
I followed how it was in your code already: https://github.com/openresty/lua-ssl-nginx-module/blob/master/src/ngx_http_lua_ssl_module.c#L229
Should I change it?